std_msgs/Header header
string[] labels
float32[] scores
sensor_msgs/Image segmasks
